scripts/installUbuntuDeps.sh

```diff
@@ -157,16 +157,12 @@ install_mediadeps(){
   sudo apt-get -qq install yasm libvpx. libx264.
   if [ -d $LIB_DIR ]; then
     cd $LIB_DIR
-    if [ ! -f ./v11.1.tar.gz ]; then
```

Comment:
We need this check also for ffmpeg to make sure we download and compile this when it's already there